.github/workflows/deploy.yml

```
@@ -7,6 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+
+    - name: Style test
```

Elsewhere we have used Lint or Linter. You may want to change it for consistency:

```suggestion
    - name: Linter
```